machine_learning/benchmarks/opflow_qnn_base_classifier_benchmark.py

```diff
@@ -0,0 +1,135 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Base for Opflow based classifier benchmarks."""
+from abc import ABC
+from typing import Optional, Tuple
+
+import numpy as np
+from qiskit.algorithms.optimizers import Optimizer
+from qiskit.circuit.library import RealAmplitudes, ZZFeatureMap
+from qiskit.utils import algorithm_globals
+from qiskit_machine_learning.algorithms import NeuralNetworkClassifier
+from qiskit_machine_learning.neural_networks import TwoLayerQNN
+from sklearn.datasets import load_iris
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler
+
+from .base_classifier_benchmark import BaseClassifierBenchmark
+from .datasets import (
+    DATASET_SYNTHETIC_CLASSIFICATION_FEATURES,
+    DATASET_SYNTHETIC_CLASSIFICATION_LABELS,
+)
+
+
+class OpflowQnnBaseClassifierBenchmark(BaseClassifierBenchmark, ABC):
+    """Base for Opflow Classifier benchmarks."""
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        # prepare synthetic
+        (
+            synth_train_features,
+            synth_test_features,
+            synth_train_labels,
+            synth_test_labels,
+        ) = train_test_split(
+            DATASET_SYNTHETIC_CLASSIFICATION_FEATURES,
+            DATASET_SYNTHETIC_CLASSIFICATION_LABELS,
+            test_size=5,
+            shuffle=False,
+        )
+
+        # in {-1, +1} for opflow
+        synth_train_labels = 2 * synth_train_labels - 1
+        synth_test_labels = 2 * synth_test_labels - 1
+
+        # prepare iris
```

Comment:
```suggestion
```
Code is self-explanatory imo.

Reply:
>         # prepare iris

> Code is self-explanatory imo.

This is for consistency with another parts of the code. I'd prefer to keep.